modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,72 +3,146 @@
 // * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
 
 :_content-type: PROCEDURE
-[id="ztp-adding-new-content-to-gitops-ztp_{context}"]
-= Adding new content to the {ztp} pipeline
 
-The source CRs in the {ztp-first} site generator container provide a set of critical features and node tuning settings for RAN Distributed Unit (DU) applications. These are applied to the clusters that you deploy with {ztp}. To add or modify existing source CRs in the `ztp-site-generate` container, rebuild the `ztp-site-generate` container and make it available to the hub cluster, typically from the disconnected registry associated with the hub cluster. Any valid {product-title} CR can be added.
+[id="ztp-adding-new-content-to-gitops-ztp_{context}"]
+= Adding custom content to the {ztp} pipeline
 
 Perform the following procedure to add new content to the {ztp} pipeline.
 
 .Procedure
 
-. Create a directory containing a Containerfile and the source CR YAML files that you want to include in the updated `ztp-site-generate` container, for example:
+. Create a subdirectory,  `source-crs`,  in the directory containing the `kustomization.yaml` file for the `PolicyGenTemplate` CR. 
+
+. Add your custom CRs to this folder, as shown in the following example:
 +
 [source,text]
 ----
-ztp-update/
-├── example-cr1.yaml
-├── example-cr2.yaml
-└── ztp-update.in
+example
+└── policygentemplates
+    ├── dev.yaml
+    ├── kustomization.yaml
+    ├── mec-edge-sno1.yaml
+    ├── sno.yaml
+    └── source-crs <1>
+        ├── PaoCatalogSource.yaml
+        ├── PaoSubscription.yaml
+        ├── bell
+        |   ├── apiserver-config.yaml
+        |   └── disable-nic-lldp.yaml
+        └── elasticsearch
+            ├── ElasticsearchNS.yaml
+            └── ElasticsearchOperatorGroup.yaml
 ----
+<1> The `sources-crs` subdirectory is in the same directory as the `kustomization.yaml` file.
 
-. Add the following content to the `ztp-update.in` Containerfile:
-+
-[source,text,subs="attributes+"]
+. Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `custom-crs` directory.
```

Comment:
There is no custom-crs directory here, either in the custom directory under source-crs or mention source-crs/custom directory.

Reply:
ok